.github/workflows/python-package.yml

```diff
@@ -0,0 +1,45 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
+
+on:
+  push:
+    branches: [ master, dev ]
+  pull_request:
+    branches: [ master, dev ]
+  schedule:
+  - cron: "0 2 * * 0-6"
```

Comment:
Is it really necessary to test every day? Nothing actually changes most days, and that'll add up to a lot of CO2.

Reply:
Done.